DlgSetting.h

```diff
@@ -14,16 +14,14 @@ class _PageInfo
 };
 namespace autoresize
 {
-	enum ResizeSpecifier
-	{
-		LEFT = 0x01,
-		RIGHT = 0x02,
-		TOP = 0x04,
-		BOTTOM = 0x08,
-		NO_HOR_RESIZE = 0x10,
-		NO_VERT_RESIZE = 0x20,
-	};
-
+	//ResizeSpecifier definitions
+	const UINT LEFT = 0x01;

```

Comment:
enum or scoped enum(enum class) is not suitable for use as flag values, so I have changed these into simple variables.

I'm wondering whether add prefix like `RESIZE_SPECOFIER_LEFT`.